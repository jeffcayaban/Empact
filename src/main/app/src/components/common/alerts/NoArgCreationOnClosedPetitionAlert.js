import React from 'react'
import { Alert } from "react-bootstrap";

/**
 * Is the alert for when an argument can not be created on a closed petition.
 */

class NoArgCreationOnClosedPetitionAlert extends React.PureComponent {
    render() {
        return (
            <div className={"secondaryPageContainerStyle"}>
                <Alert bsStyle={"danger"}>
                    <h4>You cannot create an argument on a closed petition.</h4>
                </Alert>
            </div>
        );
    }
}

export default NoArgCreationOnClosedPetitionAlert;