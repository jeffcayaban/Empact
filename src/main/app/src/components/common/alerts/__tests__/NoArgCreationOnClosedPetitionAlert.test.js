import React from 'react';
import toJson from "enzyme-to-json";
import { shallow } from "enzyme";
import NoArgCreationOnClosedPetitionAlert from "../NoArgCreationOnClosedPetitionAlert";

describe('NoArgCreationOnClosedPetitionAlert', () => {

    it('should render successfully', () => {
        const component = shallow(<NoArgCreationOnClosedPetitionAlert />);
        expect(toJson(component)).toMatchSnapshot();
    });
});
