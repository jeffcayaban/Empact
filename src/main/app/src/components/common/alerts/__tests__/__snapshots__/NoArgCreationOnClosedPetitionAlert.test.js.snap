// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoArgCreationOnClosedPetitionAlert should render successfully 1`] = `
<div
  className="secondaryPageContainerStyle"
>
  <Alert
    bsClass="alert"
    bsStyle="danger"
    closeLabel="Close alert"
  >
    <h4>
      You cannot create an argument on a closed petition.
    </h4>
  </Alert>
</div>
`;
